### PR TITLE
replace strdup elektraStrDup

### PR DESCRIPTION
**Pull request by markus2330**
# Purpose

replace strdup elektraStrDup

# Checklist

Check relevant points but please do not remove entries.
For docu fixes, spell checking, and similar nothing
needs to be checked.

- [ ] commit messages are fine ("module: short statement" syntax and refer to issues)
- [ ] I added unit tests
- [ ] I ran all tests locally and everything went fine
- [ ] affected documentation is fixed
- [ ] I added code comments, logging, and assertions (see doc/CODING.md)
- [ ] meta data is updated (e.g. README.md of plugins)
- [ ] release notes are updated (doc/news/_preparation_next_release.md)

### CODE REVIEW

File: doc/man/man1/kdb-elektrify-getenv.1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .\" generated with Ronn/v0.7.3
 .\" http://github.com/rtomayko/ronn/tree/0.7.3
 .
-.TH "KDB\-ELEKTRIFY\-GETENV" "1" "February 2018" "" ""
```
Comment (sanssecours) [inline doc/man/man1/kdb-elektrify-getenv.1]: Please do not add those pointless changes. If you want to get rid of these files, below is a Fish script I use to do just that:

```fish
for file in (git ls-files -m)
	if echo "$file" | egrep -q '\.[17]$' && 
	   test (git diff --numstat "$file" | cut -f1) -le 1 && 
	   git diff "$file" | grep -q '^-.TH';
		git checkout "$file"
	end
end
```

.

Comment (markus2330) [reply doc/man/man1/kdb-elektrify-getenv.1]: Thank you for pointing that out. The script, however, did not work for me:

```
Expected end of the statement, but instead found a '&'
scripts/git-checkout-ronn-files (line 4):       if echo "$file" | egrep -q '\.[17]$' && 
                                                                                      ^
<W> fish: Error while reading file scripts/git-checkout-ronn-files
```

fish, version 2.4.0

Comment (sanssecours) [reply doc/man/man1/kdb-elektrify-getenv.1]: Yeah, it requires the latest unreleased version of Fish (3.0) 😊. Below is a `bash` version of the same script:

```bash
#!/usr/bin/env bash

for file in $(git ls-files -m); do
	if echo "$file" | egrep -q '\.[17]$' && \
	   test $(git diff --numstat "$file" | cut -f1) -le 1 && \
	   git diff "$file" | grep -q '^-.TH'; then
		git checkout "$file"
	fi
done
```

.

**Comment by markus2330** [comment]
Thank you for the review!

**Pull request merged**